--- libgnomeui-2.24.5/libgnomeui/gnome-thumbnail.c~	2010-03-29 10:48:34.000000000 +0000
+++ libgnomeui-2.24.5/libgnomeui/gnome-thumbnail.c	2014-12-31 18:36:43.725812339 +0000
@@ -863,7 +863,11 @@
     }
   close (tmp_fd);
   
+#ifdef __ILP32__
+  g_snprintf (mtime_str, 21, "%lld",  original_mtime);
+#else
   g_snprintf (mtime_str, 21, "%ld",  original_mtime);
+#endif
   width = gdk_pixbuf_get_option (thumbnail, "tEXt::Thumb::Image::Width");
   height = gdk_pixbuf_get_option (thumbnail, "tEXt::Thumb::Image::Height");
 
@@ -970,7 +970,11 @@
     }
   close (tmp_fd);
   
+#ifdef __ILP32__
+  g_snprintf (mtime_str, 21, "%lld",  mtime);
+#else
   g_snprintf (mtime_str, 21, "%ld",  mtime);
+#endif
   pixbuf = gdk_pixbuf_new (GDK_COLORSPACE_RGB, TRUE, 8, 1, 1);
   saved_ok  = gdk_pixbuf_save (pixbuf,
 			       tmp_path,
