diff -aurN libgnomeui-2.8.0.orig/libgnomeui/gnome-gconf-ui.c libgnomeui-2.8.0/libgnomeui/gnome-gconf-ui.c
--- libgnomeui-2.8.0.orig/libgnomeui/gnome-gconf-ui.c	2002-05-31 20:54:32.000000000 +0200
+++ libgnomeui-2.8.0/libgnomeui/gnome-gconf-ui.c	2004-10-09 11:33:17.371085656 +0200
@@ -148,6 +148,8 @@
         g_return_val_if_fail(eid.client != NULL, FALSE);
         g_return_val_if_fail(pending_errors != NULL, FALSE);
 
+        GDK_THREADS_ENTER();
+
         if (current_dialog == NULL) {
                 GtkWidget *dialog;
                 gboolean have_overridden = FALSE;
@@ -244,6 +246,8 @@
         eid.client = NULL;
 
         gtk_window_present (GTK_WINDOW (current_dialog));
+
+        GDK_THREADS_LEAVE();
         
         return FALSE;
 }
diff -aurN libgnomeui-2.8.0.orig/libgnomeui/gnome-ice.c libgnomeui-2.8.0/libgnomeui/gnome-ice.c
--- libgnomeui-2.8.0.orig/libgnomeui/gnome-ice.c	2002-02-21 22:57:01.000000000 +0100
+++ libgnomeui-2.8.0/libgnomeui/gnome-ice.c	2004-10-09 11:33:17.374085200 +0200
@@ -54,6 +54,8 @@
   IceConn connection = (IceConn) data;
   IceProcessMessagesStatus status;
 
+  GDK_THREADS_ENTER();
+
   status = IceProcessMessages (connection, NULL, NULL);
 
   if (status == IceProcessMessagesIOError)
@@ -74,6 +76,8 @@
 	}
     }
 
+  GDK_THREADS_LEAVE();
+
   return TRUE;
 }
 
diff -aurN libgnomeui-2.8.0.orig/libgnomeui/gnome-icon-sel.c libgnomeui-2.8.0/libgnomeui/gnome-icon-sel.c
--- libgnomeui-2.8.0.orig/libgnomeui/gnome-icon-sel.c	2004-02-11 17:46:20.000000000 +0100
+++ libgnomeui-2.8.0/libgnomeui/gnome-icon-sel.c	2004-10-09 11:33:17.372085504 +0200
@@ -353,6 +353,8 @@
 		return FALSE;
 	}
 
+	GDK_THREADS_ENTER();
+
 	append_an_icon (gis, list->data);
 
 	g_free (list->data);
@@ -368,6 +370,8 @@
 
 	gis->_priv->load_i ++;
 
+	GDK_THREADS_LEAVE();
+
 	return TRUE;
 }
 
diff -aurN libgnomeui-2.8.0.orig/libgnomeui/gnome-pixmap-entry.c libgnomeui-2.8.0/libgnomeui/gnome-pixmap-entry.c
--- libgnomeui-2.8.0.orig/libgnomeui/gnome-pixmap-entry.c	2004-02-10 21:15:13.000000000 +0100
+++ libgnomeui-2.8.0/libgnomeui/gnome-pixmap-entry.c	2004-10-09 11:33:17.373085352 +0200
@@ -237,6 +237,8 @@
 {
 	GSList *li,*tmp;
 
+	GDK_THREADS_ENTER();
+
 	tmp = changed_pentries;
 	changed_pentries = NULL;
 	if(tmp) {
@@ -248,6 +250,8 @@
 	}
 	change_timeout = 0;
 
+	GDK_THREADS_LEAVE();
+
 	return FALSE;
 }
 
