diff -urN libgnomeui-2.18.0/configure.in libgnomeui-2.18.0.new/configure.in
--- libgnomeui-2.18.0/configure.in	2007-03-12 13:39:22.000000000 +0100
+++ libgnomeui-2.18.0.new/configure.in	2007-03-13 09:59:26.000000000 +0100
@@ -140,7 +140,9 @@
 dnl
 
 GNOME_HAVE_SM=true
-PKG_CHECK_MODULES(SM, sm ice, :, [
+PKG_CHECK_MODULES(SM, sm ice, [
+AC_DEFINE(HAVE_LIBSM, 1, [Define to 1 if you have libSM installed])
+], [
 # pkg-config modules not found (only present since X11R7); use old-style
 # detection
 AC_PATH_XTRA
