--- libgnomeui-1.117.0/configure.in~	Thu May 23 08:01:33 2002
+++ libgnomeui-1.117.0/configure.in	Thu May 23 08:01:33 2002
@@ -19,15 +19,6 @@
 ALL_LINGUAS="az de en_GB el es et fi fr hu it ja ko lt ms nn no pl pt pt_BR ru sk sl sv ta tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-AC_SUBST(LDFLAGS)
-
-GNOME_COMPILE_WARNINGS
-
-dnl Define GNOME_ENABLE_DEBUG if the --enable-debug switch was given.
-GNOME_DEBUG_CHECK
-
 dnl
 dnl Check for -lSM (for gnome-client)
 dnl
