diff -urN aa/libgnomeui-2.5.90/libgnomeui/gnome-file-entry.c libgnomeui-2.5.90/libgnomeui/gnome-file-entry.c
--- aa/libgnomeui-2.5.90/libgnomeui/gnome-file-entry.c	2004-02-10 21:15:13.000000000 +0100
+++ libgnomeui-2.5.90/libgnomeui/gnome-file-entry.c	2004-03-03 14:13:52.268446928 +0100
@@ -557,8 +557,10 @@
 			if (fw->window)
 				gdk_window_raise (fw->window);
 
-			gtk_file_chooser_set_folder_mode (GTK_FILE_CHOOSER (fw),
-							  fentry->_priv->directory_entry);
+			if (fentry->_priv->directory_entry)
+				gtk_file_chooser_set_action (GTK_FILE_CHOOSER (fw), GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER);
+			else
+				gtk_file_chooser_set_action (GTK_FILE_CHOOSER (fw), GTK_FILE_CHOOSER_ACTION_OPEN);
 
 			p = build_filename (fentry);
 			if (p) {
@@ -589,6 +591,13 @@
 
 	/* doesn't exist, create it */
 	if (fentry->_priv->use_filechooser) {
+		GtkFileChooserAction action;
+
+		if (fentry->_priv->directory_entry)
+			action = GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER;
+		else
+			action = GTK_FILE_CHOOSER_ACTION_OPEN;
+
 		fentry->fsw = gtk_file_chooser_dialog_new (fentry->_priv->browse_dialog_title
 							   ? fentry->_priv->browse_dialog_title
 							   : _("Select file"),
@@ -601,9 +610,6 @@
 
 		gtk_dialog_set_default_response (GTK_DIALOG (fw), GTK_RESPONSE_ACCEPT);
 
-		gtk_file_chooser_set_folder_mode (GTK_FILE_CHOOSER (fw),
-						  fentry->_priv->directory_entry);
-
 		p = build_filename (fentry);
 		if (p) {
 			gtk_file_chooser_set_filename (GTK_FILE_CHOOSER (fw), p);
