diff -urN aa/libgnomeui-2.5.2/libgnomeui/gnome-authentication-manager.c libgnomeui-2.5.2/libgnomeui/gnome-authentication-manager.c
--- aa/libgnomeui-2.5.2/libgnomeui/gnome-authentication-manager.c	2003-12-05 14:20:08.000000000 +0100
+++ libgnomeui-2.5.2/libgnomeui/gnome-authentication-manager.c	2004-01-12 19:35:16.516195288 +0100
@@ -283,7 +283,7 @@
 					       in_args->flags & GNOME_VFS_MODULE_CALLBACK_FULL_AUTHENTICATION_NEED_DOMAIN);
 	gnome_password_dialog_set_show_password (dialog, 
 						 in_args->flags & GNOME_VFS_MODULE_CALLBACK_FULL_AUTHENTICATION_NEED_PASSWORD);
-	gnome_password_dialog_set_show_remember (dialog, gnome_keyring_is_availible ());
+	gnome_password_dialog_set_show_remember (dialog, gnome_keyring_is_available ());
 
 	return dialog;
 }
