diff -aurN libgnomeui-2.7.1.orig/configure.in libgnomeui-2.7.1/configure.in
--- libgnomeui-2.7.1.orig/configure.in	2004-05-31 23:54:03.000000000 +0200
+++ libgnomeui-2.7.1/configure.in	2004-06-05 17:16:30.477198936 +0200
@@ -75,7 +75,7 @@
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE",
                    [Name of gettext translation domain for library])
 
-ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB eo es et eu fa fi fr ga gu he hi hr hu id is it ja kn ko li lt lv mi mk ml mn mr ms ne nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta th tk tr uk vi wa yi zh_CN zh_TW"
+ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB eo es et eu fa fi fr ga gu he hi hr hu id is it ja kn ko li lt lv mi mk ml mn mr ms nb ne nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta th tk tr uk vi wa yi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 AM_GLIB_DEFINE_LOCALEDIR([GNOMEUILOCALEDIR])
 
