diff -Nru libgnomeui-2.5.1.orig/libgnomeui/gnome-password-dialog.h libgnomeui-2.5.1/libgnomeui/gnome-password-dialog.h
--- libgnomeui-2.5.1.orig/libgnomeui/gnome-password-dialog.h	2003-12-04 19:01:24.000000000 +0100
+++ libgnomeui-2.5.1/libgnomeui/gnome-password-dialog.h	2003-12-26 22:34:46.628264848 +0100
@@ -54,7 +54,7 @@
 typedef enum {
 	GNOME_PASSWORD_DIALOG_REMEMBER_NOTHING,
 	GNOME_PASSWORD_DIALOG_REMEMBER_SESSION,
-	GNOME_PASSWORD_DIALOG_REMEMBER_FOREVER,
+	GNOME_PASSWORD_DIALOG_REMEMBER_FOREVER
 } GnomePasswordDialogRemember;
 
 
